plugins/inputs/iptables/iptables.go

```diff
@@ -91,7 +91,7 @@ func (ipt *Iptables) chainList(table, chain string) (string, error) {
 	}
 	iptablesBaseArgs := "-nvL"
 	if ipt.UseLock {
-		iptablesBaseArgs = "-wnvL"
+		iptablesBaseArgs = "-w -nvL"
```

Comment:
Shouldn't this be a slice in the command args?

Reply:
Yeah